.github/workflows/microbenchmark.yml

```diff
@@ -0,0 +1,47 @@
+name: microbenchmark
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
+
+# limit the access of the generated GITHUB_TOKEN
+permissions:
+  contents: read
+
+jobs:
+  microbenchmark:
+    runs-on: ubuntu-latest
+    # wait up to 1 hour
+    timeout-minutes: 60
+    steps:
+      - id: buildkite
+        name: Run buildkite pipeline
+        uses: elastic/apm-pipeline-library/.github/actions/buildkite@current
+        with:
+          vaultUrl: ${{ secrets.VAULT_ADDR }}
+          vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
+          vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          pipeline: apm-agent-microbenchmark
+          waitFor: true
+          printBuildLogs: true
```

Comment:
The download of the logs to the GitHub Actions run output has some `\r\n` / double-spacing lines issue and is very long. It makes it fairly unhelpful to browse:

![Screenshot 2023-05-26 at 1 11 15 PM](https://github.com/elastic/apm-agent-nodejs/assets/46866/93531829-cb1a-49fc-bfc9-e58c0aa674f0)


Browsing the log output at https://buildkite.com/elastic/apm-agent-microbenchmark/builds/82 seems sufficient (and nicer) and it means we don't need to duplicate it:

![Screenshot 2023-05-26 at 1 14 01 PM](https://github.com/elastic/apm-agent-nodejs/assets/46866/b03c4e65-b1b9-4d4d-9495-b396a2196b56)

What do you think about *not* printing the build logs here?

```suggestion
          printBuildLogs: false  # click through to Buildkite to view the logs
```

A downside of this is that the Buildkite access is limited to employees.

Reply:
We will correct the bridge between Buildkite and GitHub Action to avoid this problem. The goal is to remain as much as possible within GitHub Action.